packages/libraries/cli/package.json

```diff
@@ -59,16 +59,14 @@
     "env-ci": "7.3.0",
     "git-parse": "3.0.1",
     "graphql": "^16.0.0",
-    "graphql-request": "5.1.0",
+    "@whatwg-node/fetch": "0.8.2",
     "graphql-tag": "2.12.6",
     "log-symbols": "4.1.0",
     "mkdirp": "2.1.5",
     "rimraf": "4.4.0",
-    "ts-node": "10.9.1",
```

Comment:
nowhere used for this package

Reply:
should we also remove `graphql-tag`? I think we don't really need it?